yarn-project/constants/src/constants.gen.ts

```diff
@@ -1,4 +1,4 @@
- 
+/* eslint-disable */
```

Comment:
This was added when I ran `yarn remake-constants` - I don't think this has been run on master yet and will disappear